README.md

```diff
@@ -15,34 +15,41 @@ This repo consists of:
 
 ## Usage
 
-Start with a Python environment. Install `copier` and `poetry`:
+Start with a Python environment – we typically use [conda](https://docs.anaconda.com/anaconda/install/), but any environment manager will work. Install `copier` and `poetry`. Check that you have `python` installed or updated as well.:
 
 ```bash
+conda install conda-forge::poetry
+
 python -m pip install copier
 ```
 
-Create a directory into which the project will be rendered:
+Create a repo online and make sure to choose the same license here as you will below, typically MIT. Be sure to add all the necessary team members with the right levels of permissions.
```

Comment:
I recommend just choosing `None` as the license (which leaves the repo empty), it's less likely to conflict with what the template does.